### PR TITLE
Features/ja keyboard

### PR DESCRIPTION
**Pull request by Aoblu87**
Incorrect behavior from the virtual keyboard on browser when selecting multiple languages; the language change menu does not appear; this happens with all languages.

### CODE REVIEW

**Comment by Aoblu87** [comment]
I correct the statement from the previous comment. I tested the language change on the keyboard from the browser, and it works correctly.

**Comment by Aoblu87** [comment]
To be considered the possibility that the name of the device might be entered in Japanese characters

File: keyboard.html [hunk 1]
```diff
@@ -17,6 +17,21 @@
 <div id="virtualKeyboardChromeExtension">
 	<div id="virtualKeyboardChromeExtensionDraggableLeft"></div>
 	<div id="virtualKeyboardChromeExtensionDraggableRight"></div>
+	<div id="virtualKeyboardKanaKanjiConversionResults" style="display: none;">
```
Comment (volumio) [inline keyboard.html:20]: Are these keys visible only if JP keyboard is selected? This is the behaviour we need

File: keyboard.html [hunk 2]
```diff
@@ -135,4 +150,224 @@
...
 	</div>
+	<div id="virtualKeyboardChromeExtensionHiraganaKeys" style="display: none;">
+		<table cellpadding="2">
+			<tr>
+				<td class="kbdH kbdClick" _key="っ"><span>っ</span></td>
+				<td class="kbdH kbdClick" _key="ぁ"><span>ぁ</span></td>
+				<td class="kbdH kbdClick" _key="ゃ"><span>ゃ</span></td>
+				<td class="kbdH kbdClick" _key="ぱ"><span>ぱ</span></td>
+				<td class="kbdH kbdClick" _key="ば"><span>ば</span></td>
+				<td class="kbdH kbdClick" _key="だ"><span>だ</span></td>
```
Comment (volumio) [inline keyboard.html:161]: Are these visible only if JP is selected? This is the behaviour we need

File: keyboard.html (hunk 1 above)
Comment (Aoblu87) [reply keyboard.html:20]: Yes, they are visible only on japanese keyboard.

File: keyboard.html (hunk 2 above)
Comment (Aoblu87) [reply keyboard.html:161]: Yes, they are visible only on japanese keyboard.

**Pull request closed without merging**